optimade/filtertransformers/tests/test_mongo.py

```diff
@@ -0,0 +1,184 @@
+import unittest
+from optimade.filterparser import LarkParser, ParserError
+from optimade.filtertransformers.mongo import NewMongoTransformer
+
+
+class TestMongoTransformer(unittest.TestCase):
+    version = (0, 10, 0)
+    variant = 'default'
+
+    def setUp(self):
```

Comment:
There should be a `setUpClass(cls)` method as well, this will run once before any tests are run in the class and may optimize this slightly, since `setUp` is run before _every_ test in the class?

Reply:
Good point!